awesomeConfig.cmake

```diff
@@ -218,6 +219,17 @@ if(WITH_DBUS)
         message(STATUS "DBUS not found. Disabled.")
     endif()
 endif()
+
+if(WITH_XCB_ERRORS)
+    pkg_check_modules(XCB_ERRORS xcb-errors)
+    if(XCB_ERRORS_FOUND)
+        set(AWESOME_OPTIONAL_LDFLAGS ${AWESOME_OPTIONAL_LDFLAGS} ${XCB_ERRORS_LDFLAGS})
+        set(AWESOME_OPTIONAL_INCLUDE_DIRS ${AWESOME_OPTIONAL_INCLUDE_DIRS} ${XCB_ERRORS_INCLUDE_DIRS})
+    else()
+        set(WITH_XCB_ERRORS OFF)
+        message(STATUS "xcb-errors not found. Disabled.")
+    endif()
+endif()
```

Comment:
Could this default to AUTO, and cause an error if ON was used, but it is not available?

Reply:
There is no `AUTO` for `option()`. It is a `TRUE` / `FALSE` thing. I briefly looked into it (it is possible, just not with `option()`) and then decided to just copy what we do for DBus, since apparently that was good enough so far.

If you disagree, I can look into implementing `AUTO` for `WITH_DBUS`, `GENERATE_MANPAGES`, `GENERATE_DOC`, and anything else that needs this. But I'd do that in a separate PR.

Reply:
Sure, just a quick idea.
I think it would be nice in general, but not really necessary.